src/command_modules/azure-cli-iotcentral/azure/cli/command_modules/iotcentral/_help.py

```diff
@@ -25,10 +25,10 @@
         - name: Create an IoT Central application with the free pricing tier F1, in the region of the resource group.
           text: >
             az iotcentral app create --resource-group MyResourceGroup --name MyApp --subdomain myapp
-        - name: Create an IoT Central application with the standard pricing tier S1 in the 'westus' region.
+        - name: Create an IoT Central application with the standard pricing tier S1 in the 'westus' region, based on the iotc-demo template.
           text: >
             az iotcentral app create --resource-group MyResourceGroup --name MyApp --sku S1 --location westus
-            --subdomain myapp
+            --subdomain myapp --template iotc-demo@1.0.0
```

Comment:
should we also provide a sub command to actually list available templates?

Reply:
So, Ro and I talked about that. Ultimately, templates will be a dynamic list from the template marketplace. Not only will this mean there are too many templates to list in the CLI reasonably, but also listing these would require a server call.